src/Tasks.UnitTests/ResolveSDKReference_Tests.cs

```diff
@@ -4202,6 +4203,27 @@ public void CheckDefaultingOfTargetConfigAndArchitecture()
             }
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void VerifyPlatformAliasesWork()
+        {
+            SDKReference reference = new(new TaskItem("sdkReference", new Dictionary<string, string>() { { SDKManifest.Attributes.TargetPlatform, "UAP" } }), "sdkName", "1.0.2");
+            reference.Resolve(
+                new Dictionary<string, ITaskItem>() { { "sdkName, Version=1.0.2", new TaskItem(Path.GetTempFileName(), new Dictionary<string, string>() { { "PlatformVersion", "1.0.2" } }) } },
+                "Release",
+                "x64",
+                new HashSet<string>() { "sdkName" },
+                treatErrorsAsWarnings: false,
+                prefer32Bit: false,
+                "windows",
+                new Version("1.0.2"),
+                "projectName",
+                enableMaxPlatformVersionEmptyWarning: true);
+
+            reference.ResolutionErrors.ShouldBeEmpty();
+            reference.ResolutionWarnings.ShouldBeEmpty();
```

Comment:
I don't think I understand this test and it doesn't look like the other tests. Can you clarify what it's testing?

Reply:
If it understands the idea of aliases, it should be able to resolve the SDK reference without warnings or errors. This is a pretty barebones version, so most of the parameters end up taking default values, but it hits the one that led to this particular issue.

Reply:
Can you call out via comment in the test code the relevant features of the test? That way it's less mysterious if it starts failing. Like here I think it's that in line 4210 UAP isn't a "real" platform but it should be aliased to "Windows" anyway?

Also, is there an assert you can apply to the resolution of the alias?